app/settings/requires-elevation/admin/template.hbs

```diff
@@ -143,7 +143,7 @@
             </ul>
           {{else}}
             <button {{action 'showOtpRecoveryCodes'}} class="btn btn-primary" type="submit">
-              Show backup codes
+              Show Backup Codes
```

Comment:
@chasballew you might want to also update the labels here https://github.com/aptible/dashboard.aptible.com/blob/dccac9f41bb4a2c978f4541f7cbeacb7f4642860/app/settings/requires-elevation/admin/controller.js#L48-L61 for this page. let me know if you'd like me to take care of it.

Cheers,